src/pages/post-feed/usersDummyData.ts

```diff
@@ -0,0 +1,51 @@
+export interface User {
+  id: number;
+  username: string;
+  userRank: string;
+  userPhoto: string;
+  text: string;
+  photo: string;
+}
+
+export const users: User[] = [
```

Comment:
I think you mean posts here and not users, right? 
the post is the main data we have, and the post will contain the user data in it

Reply:
Nope, I mean users cuz these data are related to users.

Reply:
yes they are related to users, but what you are showing her is posts, not users
the user is just the user data (username, image, email, ...etc) but the user will not have posts inside of it, otherwise, this would become very big, image facebook or Instagram, each user has thousands of posts, maybe even more, if we put the posts inside the user how big the user would be?